.remove-watcher.js

```diff
@@ -0,0 +1,16 @@
+var fs = require('fs')
+var files = fs.readdirSync('tests/')
+
+
+files.forEach(file => {
+  fs.readFile('tests/' + file, 'utf8', function (err, data) {
+    if (err) {
+      return console.log(err)
+    }
+    var result = data.replace(/@watch/g, '')
```

Comment:
We should really replace SPACE@watch as well, otherwise spaces will remain in the desc, something like `var result = data.replace(/ @watch/g, '').replace(/@watch/g, '')`